edx_proctoring/templates/proctored_exam/entrance.html

```diff
@@ -7,7 +7,7 @@
   </h3>
   <p>
     {% blocktrans %}
-      To be eligible to earn credit for this course, you must take and pass the proctoring review for this exam.
+      To be eligible for course credit or for a MicroMasters credential, you must pass the proctoring review for this exam.
```

Comment:
I considered that, but then saw that it is hard-coded in other strings in edX platform. It is an interesting point that this message is very edX-specific now, which isn't ideal for an open source library. @dianakhuang do you have any recommendations?